file-sink/README.md

```diff
@@ -0,0 +1,35 @@
+Spring Cloud Stream File Sink
+=============================
```

Comment:
Please use asciidoc, pretty please. 

In all cases, it's better to avoid this underlining form for titles, very hard to deal with in an automatic fashion